extractor.go

```diff
@@ -61,35 +78,70 @@ func (te *Extractor) Extract(reader io.Reader) error {
 	return nil
 }
 
-// outputPath returns the path at whicht o place tarPath
-func (te *Extractor) outputPath(tarPath string) string {
-	elems := strings.Split(tarPath, "/") // break into elems
-	elems = elems[1:]                    // remove original root
+// Sanitize toggles output sanitation, using built in sanitation functions
+// (Modify paths to be platform legal, symlinks may not escape extraction root)
+func (te *Extractor) Sanitize(toggle bool) {
```

Comment:
this input arg seems to be ignored

Reply:
Now it should actually check it and unset the function pointers on `false`.